src/forum-tool-selector-settings/ForumToolSelectorSettings.jsx

```diff
@@ -0,0 +1,130 @@
+  },
+  {
+    forumId: 'edX Forum1',
+    logo: 'https://cdn-blog.lawrencemcdaniel.com/wp-content/uploads/2018/01/22125436/edx-logo.png',
+    description: 'Start conversations with other learners, ask questions, and interact with other learners in the course.',
+    supportLevel: 'Full support',
+    isAvailable: true,
+    features: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'Embedded Course Units', 'WCAG 2.1 Support'],
+    allfeatures: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'Embedded Course Units', 'WCAG 2.1 Support'],
+  },
+  {
+    forumId: 'Piazza1',
+    logo: 'https://cdn-blog.lawrencemcdaniel.com/wp-content/uploads/2018/01/22125436/edx-logo.png',
+    description: 'Piazza is designed to connect students, TAs, and professors so every student can get the help they need when they need it',
+    supportLevel: 'Partial support',
+    isAvailable: true,
+    features: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'WCAG 2.1 Support'],
+    allfeatures: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'Embedded Course Units', 'WCAG 2.1 Support'],
+  },
+  {
+    forumId: 'Yellowdig1',
+    logo: 'https://cdn-blog.lawrencemcdaniel.com/wp-content/uploads/2018/01/22125436/edx-logo.png',
+    description: 'Yellowdig is the digital solution that impacts the entire student lifecycle and enables lifelong learning.',
+    supportLevel: 'Coming soon',
+    isAvailable: false,
+    features: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'WCAG 2.1 Support'],
+    allfeatures: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'Embedded Course Units', 'WCAG 2.1 Support'],
+  },
+  {
+    forumId: 'edX Forum2',
+    logo: 'https://cdn-blog.lawrencemcdaniel.com/wp-content/uploads/2018/01/22125436/edx-logo.png',
+    description: 'Start conversations with other learners, ask questions, and interact with other learners in the course.',
+    supportLevel: 'Full support',
+    isAvailable: true,
+    features: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'Embedded Course Units', 'WCAG 2.1 Support'],
+    allfeatures: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'Embedded Course Units', 'WCAG 2.1 Support'],
+  },
+  {
+    forumId: 'Piazza2',
+    logo: 'https://cdn-blog.lawrencemcdaniel.com/wp-content/uploads/2018/01/22125436/edx-logo.png',
+    description: 'Piazza is designed to connect students, TAs, and professors so every student can get the help they need when they need it',
+    supportLevel: 'Partial support',
+    isAvailable: true,
+    features: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'WCAG 2.1 Support'],
+    allfeatures: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'Embedded Course Units', 'WCAG 2.1 Support'],
```

Comment:
I don't think this will be a thing found on every forum entry. It can however be derived from all the forums. Remove this form here, this will be handled by the reducer.